src/elements/content-preview/ContentPreview.js

```diff
@@ -1145,15 +1145,18 @@ class ContentPreview extends React.PureComponent<Props, State> {
         });
     };
 
-    handleAnnotationSelect = ({
-        file_version: { id: annotationFileVersionId },
-        target: { location = {} },
-    }: Annotation) => {
+    handleAnnotationSelect = (annotation: Annotation) => {
+        const {
+            file_version: { id: annotationFileVersionId },
+            target,
+        } = annotation;
+        const { location = {} } = target;
         const { selectedVersion } = this.state;
```

Comment:
Hmm looks like I goofed with this destructuring from `this.state` twice, would you mind fixing that for me?

Reply:
No problem, surprised I didn't catch it 😱 .